mscore/plugin/pluginCreator.cpp

```diff
@@ -590,5 +601,27 @@ void PluginCreator::showManual()
             }
       manualDock->setVisible(!manualDock->isVisible());
       }
-}
+
```

Comment:
Coming from qmlEdit.cpp
It's the whole window that should change MuseScore's state, not just the textarea

Reply:
Required for the shortcuts.